<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Дилер (База данных) 1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButtonPrint">
      <property name="text">
       <string>Печать</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButtonUpdate">
      <property name="text">
       <string>Обновить</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>521</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButtonQuit">
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpenDB"/>
    <addaction name="actionCreateDB"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuDB">
    <property name="title">
     <string>База данных</string>
    </property>
    <widget class="QMenu" name="menuSale">
     <property name="title">
      <string>Информация о продажах</string>
     </property>
     <addaction name="actionContractsSale"/>
     <addaction name="actionChecks"/>
     <addaction name="actionProductsSale"/>
    </widget>
    <widget class="QMenu" name="menuSupply">
     <property name="title">
      <string>Информация о поставках</string>
     </property>
     <addaction name="actionContractsSupply"/>
     <addaction name="actionPayments"/>
     <addaction name="actionProductsSupply"/>
    </widget>
    <addaction name="actionVendors"/>
    <addaction name="actionClients"/>
    <addaction name="separator"/>
    <addaction name="actionProductTypes"/>
    <addaction name="actionProducts"/>
    <addaction name="separator"/>
    <addaction name="menuSale"/>
    <addaction name="menuSupply"/>
   </widget>
   <widget class="QMenu" name="menuQuery">
    <property name="title">
     <string>Запросы</string>
    </property>
    <addaction name="actionQueryProducts"/>
    <addaction name="actionQueryClients"/>
    <addaction name="actionQueryPotentialClients"/>
    <addaction name="actionQuerySaleSupplyDone"/>
    <addaction name="actionQueryAllProducts"/>
    <addaction name="actionQueryPaid"/>
    <addaction name="actionQueryUnpaid"/>
    <addaction name="actionQuerySaleSupplyWait"/>
    <addaction name="actionQueryContracts"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuDB"/>
   <addaction name="menuQuery"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenDB">
   <property name="text">
    <string>Открыть базу данных</string>
   </property>
  </action>
  <action name="actionCreateDB">
   <property name="text">
    <string>Создать базу данных</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionVendors">
   <property name="text">
    <string>Производители</string>
   </property>
  </action>
  <action name="actionClients">
   <property name="text">
    <string>Клиенты</string>
   </property>
   <property name="toolTip">
    <string>Клиенты</string>
   </property>
  </action>
  <action name="actionProductTypes">
   <property name="text">
    <string>Типы товаров</string>
   </property>
  </action>
  <action name="actionProducts">
   <property name="text">
    <string>Товары</string>
   </property>
  </action>
  <action name="actionContractsSale">
   <property name="text">
    <string>Договоры продаж</string>
   </property>
  </action>
  <action name="actionContractsSupply">
   <property name="text">
    <string>Договоры поставок</string>
   </property>
  </action>
  <action name="actionChecks">
   <property name="text">
    <string>Счета продаж</string>
   </property>
  </action>
  <action name="actionPayments">
   <property name="text">
    <string>Платёжные поручения поставок</string>
   </property>
  </action>
  <action name="actionProductsSale">
   <property name="text">
    <string>Продажи товаров</string>
   </property>
  </action>
  <action name="actionProductsSupply">
   <property name="text">
    <string>Поставки товаров</string>
   </property>
  </action>
  <action name="actionQueryAllProducts">
   <property name="text">
    <string>Товары на складе</string>
   </property>
  </action>
  <action name="actionQueryProducts">
   <property name="text">
    <string>Номенклатура продукции</string>
   </property>
  </action>
  <action name="actionQueryClients">
   <property name="text">
    <string>Список клиентов по типу товаров</string>
   </property>
  </action>
  <action name="actionQueryPotentialClients">
   <property name="text">
    <string>Список потенциальных клиентов</string>
   </property>
  </action>
  <action name="actionQuerySaleSupplyDone">
   <property name="text">
    <string>Поставленные и проданные товары</string>
   </property>
  </action>
  <action name="actionQueryPaid">
   <property name="text">
    <string>Оплаченные счета</string>
   </property>
  </action>
  <action name="actionQueryUnpaid">
   <property name="text">
    <string>Неоплаченные счета</string>
   </property>
  </action>
  <action name="actionQuerySaleSupplyWait">
   <property name="text">
    <string>Заказанные товары</string>
   </property>
  </action>
  <action name="actionQueryContracts">
   <property name="text">
    <string>Договоры</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonQuit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>750</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
